pkg/installer/upgrade.go

```diff
@@ -34,7 +34,7 @@ func Upgrade(uninstallConfig *apiv1.KubectlStorageOSConfig, installConfig *apiv1
 	}
 
 	// create (un)installer with in-mem fs of operator and cluster to be uninstalled
-	uninstaller, err := NewInstaller(uninstallConfig, false)
+	uninstaller, err := NewInstaller(uninstallConfig, false, false)
```

Comment:
for the upgrade, should we be allowing the uninstall to go ahead without knowing whether the install will be successful (ie if the min kube version exists)... seems like the check should happen before upgrade is called at all?